.jenkins

```diff
@@ -10,7 +10,7 @@ pipeline {
         ARBORX_DIR = '/opt/arborx'
         BENCHMARK_COLOR = 'no'
         BOOST_TEST_COLOR_OUTPUT = 'no'
-        CTEST_OPTIONS = '--timeout 180 --no-compress-output -T Test --test-output-size-passed=65536 --test-output-size-failed=1048576'
+        CTEST_OPTIONS = '--timeout 1800 --no-compress-output -T Test --test-output-size-passed=65536 --test-output-size-failed=1048576'
```

Comment:
I am not ok with 30min timeout

Reply:
Of course. I just wanted to make sure that there are no other errors and see how long the tests actually need. I would prefer to test with `oneDPL` in the CI independent of enabling the dependency is enabled by default and even if it might be a little slower for the test suite since using Kokkos search is covered in other configurations.

That doesn't mean that we shouldn't try to see if other search algorithms in `oneDPL` are better suited.